test/plugins/auth.test.js

```diff
@@ -485,14 +486,34 @@ describe('auth plugin test', () => {
             })
         );
 
-        it('fails to issue a jwt given an invalid application auth token', () =>
+        it('returns user signed token given an API access token', () =>
             server.inject({
+                url: `/auth/token?access_key=${apiKey}`
```

Comment:
Ohh okay. So we can merge this PR but make sure we do away with the `temporaryAccessKey` soon after.